server/src/tree-sitter/analyzer.ts

```diff
@@ -274,6 +275,50 @@ export default class Analyzer {
     (n?.type === ':' && n?.parent?.type === 'ERROR' && n?.parent?.previousSibling?.type === 'override') // when having MYVAR:append: = '123' and the second or later colon is typed
   }
 
+  public isInsidePythonRegion (
```

Comment:
Note this function is a duplicate with a function with the same name in "utils". I will clean it up once this PR or #18 is merged in order to simplify the conflicts resolution.